### PR TITLE
fix: Button icons tooltips opening out of view due to padding issue

### LINKED ISSUE #11812: [CAL-2609] /event-type-single: Button icons tooltips opening out of view due to padding issue
It looks like we've reduced the padding at the top of the event types screen so now the tooltips are out of view:

CleanShot 2023-10-11 at 05 45 14@2x

Should be:

CleanShot 2023-10-11 at 05 51 13@2x

<sub>[CAL-2609](https://linear.app/calcom/issue/CAL-2609/event-type-single-button-icons-tooltips-opening-out-of-view-due-to)</sub>

### PR DESCRIPTION
**Pull request by siddharth-movaliya**
## What does this PR do?

Fixes Button icons tooltips opening out of view due to padding issue. Set the tooltip position to bottom, hence no padding has to be changed.

Fixes #11812 
Screenshot 2023-10-12 at 10 19 51 AM

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. Create an event.
2. Open the event by clicking on it.
3. Hover on any of the buttons on the top (preview, copy link, etc). The tooltip will now open at the bottom.

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by sean-brydon — DISMISSED** [review]
Nice job. 

One small issue - can we rename "side" on button to something like tooltip Position just to make it a bit clearer as side doesnt really give any indication to what it does when used in a button without any context

**Comment by Siddharth-2382** [comment]
Yeah I thought after I finished that... nevermind will do it and ping you if it's fine?

**Comment by Siddharth-2382** [comment]
Done 👍🏻

File: packages/ui/components/button/Button.tsx [hunk 1]
```diff
@@ -213,19 +215,27 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
       {element}
     </Link>
   ) : (
-    <Wrapper data-testid="wrapper" tooltip={props.tooltip}>
+    <Wrapper data-testid="wrapper" tooltip={props.tooltip} tooltipSide={props.tooltipSide}>
```
Comment (sean-brydon) [inline packages/ui/components/button/Button.tsx:218]:
```suggestion
    <Wrapper data-testid="wrapper" tooltip={props.tooltip} tooltipSide={tooltipSide}>
```

**Comment by sean-brydon** [comment]
Thanks for that bud - i just notice the tooltips are right against the button - we could add a tooltip Offset prop to handle this. 

I tried pushing to this branch for for some reason it didnt let me 

If you dont mind would you mind cherrypicking / implementing these changes i have here 

https://github.com/calcom/cal.com/commit/df1eaf69fa154c854ac3a0a08854af882bca2ff4

**Comment by Siddharth-2382** [comment]
> Thanks for that bud - i just notice the tooltips are right against the button - we could add a tooltip Offset prop to handle this.
> 
> I tried pushing to this branch for for some reason it didnt let me
> 
> If you dont mind would you mind cherrypicking / implementing these changes i have here
> 
> [df1eaf6](https://github.com/calcom/cal.com/commit/df1eaf69fa154c854ac3a0a08854af882bca2ff4)

sure

**Review by PeerRich — APPROVED** [review]
approved on behalf of @sean-brydon

**Review by sean-brydon — APPROVED** [review]
LGTM - thanks bud

**Pull request merged**